tests/tt_metal/test_utils/df/float32.hpp

```diff
@@ -20,13 +20,13 @@ class float32 {
     uint32_t uint32_data = 0;
 
    public:
-    static const size_t SIZEOF = 4;
+    static constexpr size_t SIZEOF = 4;
 
     // create from float: no rounding, just truncate
     float32(float float_num) {
         static_assert(sizeof float_num == sizeof uint32_data, "Can only support 32bit fp");
         // just move upper 16 to lower 16 (truncate)
-        uint32_data = (*reinterpret_cast<uint32_t*>(&float_num) >> 16);
```

Comment:
A little confused here, was this a bug in the original code? It doesn't seem to match the comment above, or the `<< 16` in `to_float()`

Reply:
Original code has not been used previously, and I expect that it was not bug, but effectively reduced precision from float32 to float16. I reverted it back to previous version.